pynecone/components/datadisplay/table.py

```diff
@@ -46,12 +46,18 @@ def create(
                 children.append(TableCaption.create(caption))
 
             if headers:
+                if isinstance(headers, BaseVar):
+                    headers = headers.get_field_value()
```

Comment:
I think what we want to do is change the api for `THead`, etc to support vars. The vars will be rendered as JS variables like `{state.var}` into the code. The way it's doing it now will compile the initial value in, but it won't update dynamically.

Reply:
I get you. 
Do you have an idea to fix this issue?